examples/breathe/breathe.c

```diff
@@ -0,0 +1,34 @@
+#include "ch32fun.h"
+
+// Use Pin PD4
+#define PIN_REG GPIOD
+#define PIN_VAL 0x40
+#define PIN_NUM 4
+
+#define SCALE  64
+
+int main(){
+	uint32_t lung = 1, roll = 0, volume = 1;
+	int32_t io = 1;
+
+	SystemInit();
+	RCC->APB2PCENR |= RCC_APB2Periph_GPIOD;
```

Comment:
Please add comment showing what you are causing to be hidden here.  This is a good trick to learn but not obvious that you are replacing funConfig lines.

Reply:
I added a large block of comments to the head of the file and converted to the fun version of the config and GPIO functions.